dpdata/system.py

```diff
@@ -441,19 +443,8 @@ def remove_pbc(self, protect_layer = 9):
         protect_layer : the protect layer between the atoms and the cell
                         boundary
         """
-        nframes = self.get_nframes()
-        natoms = self.get_natoms()
         assert(protect_layer >= 0), "the protect_layer should be no less than 0"
-        for ff in range(nframes):
-            tmpcoord = self.data['coords'][ff]
-            cog = np.average(tmpcoord, axis = 0)
-            dist = tmpcoord - np.tile(cog, [natoms, 1])
-            max_dist = np.max(np.linalg.norm(dist, axis = 1))
-            h_cell_size = max_dist + protect_layer
-            cell_size = h_cell_size * 2
-            shift = np.array([1,1,1]) * h_cell_size - cog
-            self.data['coords'][ff] = self.data['coords'][ff] + np.tile(shift, [natoms, 1])
-            self.data['cells'][ff] = cell_size * np.eye(3)
+        remove_pbc(self.data, protect_layer)
```

Comment:
I would recommend reverting the `remove_pbc` method in `dpdata.System`

Reply:
If this is leaved in class dpdata.System, how do I invoke it from pymatgen/molecule.py?